NAMESPACE

```diff
@@ -1,18 +1,46 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(print,cran_event_list)
+S3method(print,cran_package)
+S3method(print,cran_package_list)
 S3method(print,pkg_search_result)
+S3method(print,r_releases)
+S3method(summary,cran_event_list)
+S3method(summary,cran_package)
+S3method(summary,cran_package_list)
 S3method(summary,pkg_search_result)
+S3method(summary,r_releases)
+export("%>%")
+export(events)
+export(list_packages)
 export(more)
+export(package)
 export(pkg_search)
 export(ps)
+export(releases)
```

Comment:
We'll need to rename these to `cran_events()`, `cran_package()`, etc. or `pkg_events()` or sg. I guess `cran_events()`.

Reply:
I renamed three functions, `cran_releases()`, `cran_events()` and `cran_package()`, is it fine?